src/07-embedding/02-artists.ts

```diff
@@ -0,0 +1,151 @@
+import { ARTIST_IDS } from "./artist-ids"
```

Comment:
We wanted the 1000 most followed p1 artists — which isn't actually a query you can send to MP currently, so we pulled a list of ids from a Gravity console instead.